### DISCUSSION TITLE
I am very confused with import_skills.py

### DISCUSSION THREAD

**Discussion by selfishark** [microsoft/semantic-kernel#2696] [General]
I am new to programming (learning Python) and ChatCopilot is so interesting that I decided to start from the very beginning. 

While practising on '[semantic functions](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/semantic-functions?tabs=python)' I faced an error import_semantic_skill_from_directory which apparently is resolved on [here](https://github.com/microsoft/semantic-kernel/pull/236)  by @renrut but couldn't understand much so I navigated and landed where the import_semantic_skill_from_directory is actually defined, [here](https://github.com/microsoft/semantic-kernel/blob/python-preview-archived-dont-delete/python/semantic_kernel/kernel_extensions/import_skills.py) but then again I tried to pull this but it connected to other functions and doesn't seems wise to just download one by one just to follow the tutorial. 

So I discovered there are just some many branches that exist for the same Semantic Kernel repo. So I wonder which one is the best to follow because I tried to rebase my code and saw 41 commits thinking it would solve my issue but nope. So I am still stuck on the first tutorial. 

Can anybody tell me how to fix import_semantic_skill_from_directory and how to make sure I am pulling the most fresh repo?

@alexchaomander can this be a topic on the Semantic Kernel Office Hours?

> In the current main, kernel_extensions/import_skills.py does not exist anymore.

**Comment by alexchaomander** [comment]
Definitely make use of the `main` branch for the latest APIs for Python that matches all the documentation. There are likely several stale branches that can be deleted.

If you're using Python, make sure to always run `pip install -U semantic-kernel` to get the latest.

**Reply by selfishark** [reply]
Thanks, man I will keep that updated. - There has been a recent update by @matthewbolanos 10 hours ago and I will give it a go. By the look of it, it solves many other issues. Thank you all!
